### PR TITLE
GCI: Add Sugar ORM and basic models

### PR DESCRIPTION
**Pull request by HITGIF**
Create model classes with Sugar ORM implemented:

- Contact
    - UUID uuid
    - String name
    - List numbers
    - Map<String, String> tags

This completes the task https://codein.withgoogle.com/dashboard/task-instances/5837074580111360/

### CODE REVIEW

**Comment by garg-anuj** [comment]
@HITGIF You will need to resolve one conflict

**Comment by HITGIF** [comment]
Done

File: app/src/main/java/com/community/jboss/leadmanagement/Classes/Contact.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+    private UUID uuid;
+    private String name;
+
+    public Contact(UUID uuid, String name) {
+        this.uuid = uuid;
+        this.name = name;
+    }
+
+    public void setUuid(UUID uuid) {
```
Comment (zwliew) [inline app/src/main/java/com/community/jboss/leadmanagement/Classes/Contact.java]: I believe UUIDs should be constant. Allowing database entry UUIDs to be modified doesn't seem to be a good idea. @garg-anuj is this right?

Comment (Rishabh2801) [reply app/src/main/java/com/community/jboss/leadmanagement/Classes/Contact.java]: I think it is fine if we not declare it as constant @zwliew . Am i right @garg-anuj ?

Comment (garg-anuj) [reply app/src/main/java/com/community/jboss/leadmanagement/Classes/Contact.java]: I felt models will be too hard to be maintained by new members so more over we will structure models and will declare further tasks 😉

Comment (Rishabh2801) [reply app/src/main/java/com/community/jboss/leadmanagement/Classes/Contact.java]: True @garg-anuj . Thanks

**Pull request merged**